settings.gradle

```diff
@@ -28,7 +28,7 @@ dependencyResolutionManagement {
             library('opentelemetry-proto', 'io.opentelemetry.proto', 'opentelemetry-proto').versionRef('opentelemetry')
             version('opensearch', '1.3.8')
             library('opensearch-rhlc', 'org.opensearch.client', 'opensearch-rest-high-level-client').versionRef('opensearch')
-            version('spring', '5.3.26')
+            version('spring', '5.2.24.RELEASE')
```

Comment:
This doesn't add any new CVEs?

Reply:
I followed the suggested fix from #2492 to set it to `5.2.24.RELEASE`. 

Now looking at https://spring.io/security/cve-2023-20863, `5.3.27` should also fix the CVE. Changed to `5.3.27` instead.